aws-iam-policies/generated/restricted-policy-1.json5

```diff
@@ -8,10 +8,14 @@
                 "acm:DeleteCertificate",
                 "autoscaling:SuspendProcesses",
                 "autoscaling:UpdateAutoScalingGroup",
+                "autoscaling:DescribeAutoScalingGroups",
```

Comment:
Pls have a look at the developer section of README doc. It tells that generated folder should not be touched.